rasa/core/events/__init__.py

```diff
@@ -580,17 +580,18 @@ def apply_to(self, tracker) -> None:
 
 # noinspection PyProtectedMember
 class ReminderScheduled(Event):
-    """ Allows asynchronous scheduling of action execution.
+    """ Allows asynchronous scheduling of intent / entity triggering.
```

Comment:
What is `entity triggering`?

Reply:
Well, you can trigger an intent combined with a list of entities.

Reply:
I'll formulate it more clearly.

Reply:
thanks 👍